hugo/assets/scss/components/separator.scss

```diff
@@ -10,8 +11,9 @@
     clear: both;
     display: block;
     height: 1px;
-    margin: 1rem auto;
+    margin: 0.5rem auto 1rem;
     max-width: $w-content - ($p-gutter * 2);
+    opacity: 0.2;
```

Comment:
Double check if this is working correctly because I remember the hr tag itself also has opacity because it has a rgba background set. We want this set or by opacity or by rgba, not both I think

Reply:
Just checked this. We use transparentize to set the `--hr-color`. I don't know if it works on the `hr` but it doesn't changes anything on the separator. Opacity does lighten the separator. How do you want this fixed?

Reply:
I thought separator was set on an hr, but it seems it's a div. In that case this is fine. Sorry for the confusion :sweat_smile: